datm/datm_datamode_clmncep_mod.F90

```diff
@@ -61,7 +58,6 @@ module datm_datamode_clmncep_mod
   real(r8), pointer :: strm_swdn(:)      => null()
   real(r8), pointer :: strm_swdndf(:)    => null()
   real(r8), pointer :: strm_swdndr(:)    => null()
-  real(r8), pointer :: strm_prec(:)      => null()
```

Comment:
We are moving away from having precc and precl - so I think its premature to remove prec. This was an issue that Erik raised and that we should add as an issue in cdeps as well as cesm.

Reply:
This variable is not used anywhere in this module and can safely be removed.